Cargo.toml

```diff
@@ -10,6 +10,9 @@ edition = "2021"
 repository = "https://github.com/stackabletech/listener-operator"
 
 [workspace.dependencies]
+stackable-operator = { git = "https://github.com/stackabletech/operator-rs.git", tag = "stackable-operator-0.89.1" }
+stackable-telemetry = { git = "https://github.com/stackabletech/operator-rs.git", tag = "stackable-telemetry-0.4.0" }
```

Comment:
I suspect this is going to go wrong as soon as you add any common dependencies in op-rs between them (like stackable-shared?).

Reply:
Currently, there is no cross-dependency between the two, but with an upcoming change (see https://github.com/stackabletech/operator-rs/pull/1001), there will be one.

We can think about re-exporting `stackable-telemetry` in `stackable-operator` (behind a feature flag) if we want. Something like `stackable_operator::telemetry` comes to mind.